glustercli/cmd/volume-create.go

```diff
@@ -54,10 +70,73 @@ func init() {
 	volumeCreateCmd.Flags().BoolVar(&flagAllowRootDir, "allow-root-dir", false, "Allow root directory")
 	volumeCreateCmd.Flags().BoolVar(&flagAllowMountAsBrick, "allow-mount-as-brick", false, "Allow mount as bricks")
 	volumeCreateCmd.Flags().BoolVar(&flagCreateBrickDir, "create-brick-dir", false, "Create brick directory")
+
+	// Smart Volume Flags
+	volumeCreateCmd.Flags().StringVar(&flagCreateVolumeSize, "size", "", "Size of the Volume")
+	volumeCreateCmd.Flags().IntVar(&flagCreateDistributeCount, "distribute", 1, "Distribute Count")
+	volumeCreateCmd.Flags().StringVar(&flagCreateLimitPeers, "limit-peers", "", "Use bricks only from these Peers")
+	volumeCreateCmd.Flags().StringVar(&flagCreateLimitZones, "limit-zones", "", "Use bricks only from these Zones")
+	volumeCreateCmd.Flags().StringVar(&flagCreateExcludePeers, "exclude-peers", "", "Do not use bricks from these Peers")
+	volumeCreateCmd.Flags().StringVar(&flagCreateExcludeZones, "exclude-zones", "", "Do not use bricks from these Zones")
```

Comment:
Later on you convert the above 4 flags to a list. You can instead use `StringSliceVar` and have the flags package do that for you.